src/EigenVector.cpp

```diff
@@ -40,7 +40,7 @@ bool EigenVector::test_trace_sum(const ssize_t t, const bool do_throw) {
   const std::complex<double> trace = VdV.trace();
   const std::complex<double> sum = VdV.sum();
   // we allow for some deviation
-  if( abs( trace.real() - V[t].cols() ) > 2*V[t].cols()*DBL_EPSILON ){
+  if( abs( trace.real() - V[t].cols() ) > 32*V[t].cols()*DBL_EPSILON ){
```

Comment:
It's a bit worrying that this deviates so much (although some roundoff differences are expected when doing a multithreaded accumulation).  Perhaps the condition should be ```V[t].rows() * V[t].cols() * DBL_EPSILON``` to automatically float as the volume grows.

Reply:
> although some roundoff differences are expected when doing a multithreaded accumulation

this is especially true when doing aggressive optimisation

Reply:
> > although some roundoff differences are expected when doing a multithreaded accumulation
> 
> this is especially true when doing aggressive optimisation

Do we need the orthogonality check as well? ( If they were produced by sLapH-eigsys and the norm of the eigenvectors was 1 then they should be orthogonal as well) Just if it takes to much time